### PR TITLE
[Python3] #1052 ImportError: cannot import name 'main'

### PR DESCRIPTION
**Pull request by Sirfanas**
Original Issue is: [ImportError: cannot import name 'main' #1052](https://github.com/kivy/buildozer/issues/1052)

This is the fix I show on [this post](https://github.com/kivy/buildozer/issues/1052#issuecomment-611006248)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -75,6 +75,8 @@ WORKDIR ${WORK_DIR}
 COPY --chown=user:user . ${SRC_DIR}
 
 # installs buildozer and dependencies
-RUN pip3 install --user Cython==0.28.6 ${SRC_DIR}
+RUN pip3 install --user wheel Cython==0.28.6 ${SRC_DIR}
+
+RUN python3 -m pip install -U pip virtualenv
```
Comment (AndreMiras) [inline Dockerfile:80]: Thanks for the PR, and why the inconsistency one time with `pip3 install` and the other time `python3 -m pip install`?

Comment (Sirfanas) [reply Dockerfile:80]: Just a reflex I have to use `python3 -m pip`
I change the line into 'pip3'

**Comment by brvier** [comment]
I confirm also the workarround.

File: Dockerfile [hunk 2]
```diff
@@ -75,6 +75,9 @@ WORKDIR ${WORK_DIR}
 COPY --chown=user:user . ${SRC_DIR}
 
 # installs buildozer and dependencies
-RUN pip3 install --user Cython==0.28.6 ${SRC_DIR}
+RUN pip3 install --user wheel Cython==0.28.6 ${SRC_DIR}
+
+# Upgrade virtualenv and pip
+RUN pip3 install -U pip virtualenv
```
Comment (AndreMiras) [inline Dockerfile:81]: Thanks for the update. How about in-lining the entire thing?
```suggestion
RUN pip3 install --user --upgrade Cython==0.28.6 wheel pip virtualenv ${SRC_DIR}
```

Comment (Sirfanas) [reply Dockerfile]: Yeah look really better, I just build the image and compile my app, it's ok for me !

**Pull request merged**